### ISSUE TITLE
[sabredav] Auto-Creation of sabredav-user uses short username (without domain name)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#431]
```
Token: d8ed8da9e30db02b3327f7f314f93c1a (Don't modify this token!)
Version: 2.6 (04-11-2012)
PHP: 5.3.3-7 squeeze14
RCMAIL: 0.8.2
Database: mysql
SERVER: Apache/2.2.16 (Debian)
----
I.  Issue Description:
I use a roundcube installation with only one domain, so I configured roundcube 
to add the domain name during login automatically. If I enter user@domain the 
sabredav plugin correctly creates a dav-user in the database (the name of the 
user contains the domain name as expected). But, if I login in as "user" 
(letting roundcube add "@domain" automatically) the sabredav-plugin creates a 
user without the domain name. That could become a security risk, if the 
database is used in a multi-domain setup, when the same user exists in multiple 
domains.

II. Steps to reproduce the Issue:
1. Log in to roundcube with a user name without domain name (roundcube must be 
configured accordingly).
2. Have a look into the database (table 'users').

```

Original issue reported on code.google.com by `bloethg...@gmail.com` on 18 Nov 2012 at 3:23

**Comment by GoogleCodeExporter** [comment]
Original comment by `roland.l...@gmail.com` on 19 Nov 2012 at 4:44
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in sabredav plugin v4.0.

It was never a security issue! Please re-open ticket if there are still 
problems/concerns. Thanks!
```

Original comment by `roland.l...@gmail.com` on 10 Jan 2013 at 9:32
- Changed state: **Fixed**
